_includes/footer-contact.html

```diff
@@ -0,0 +1,30 @@
+<section class="container">
+  <h1>Stay In Touch!</h1>
+
+  <p>
+    You can reach us on our
+    <a href="https://dtc-innovation-slackin.herokuapp.com/" target="_blank">Slack channel</a>
+    or by
+    <a href="mailto:&#099;&#111;&#110;&#116;&#097;&#099;&#116;&#064;&#100;&#116;&#099;&#045;&#105;&#110;&#110;&#111;&#118;&#097;&#116;&#105;&#111;&#110;&#046;&#111;&#114;&#103;?subject=Hello">email</a>.
+    We will do our best to respond to you within the <em>next
+    business hours</em>&nbsp;—&nbsp;although we might be <em>on the field</em>
+    with a limited connectivity.
+  </p>
+
+  <form method="POST" action="https://dtc-innovation.us14.list-manage.com/subscribe/post?u=b0f6f0d51771df93463c7b23f&amp;id=fbe749b473" target="_blank">
+    <div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_b0f6f0d51771df93463c7b23f_fbe749b473" tabindex="-1" value=""></div>
```

Comment:
It is a form honeypot. A bot would likely fill any visible field but a human would not have it in front of their eyes, not it would be made actionable by a speech synthesis system.